### PR TITLE
fix(action): remove set-action

### PR DESCRIPTION
**Pull request by mariiatuzovska**
* remove ::set-output action
* make setting env by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +91,7 @@ See [integration.yml](.github/workflows/integration.yml) for an example of how t
 
 ## Other Usage Examples
 
-### Retrieve A Single Secret Without Setting Env File
+### Set Output to Job Scope Using Environment Variables
```
Comment (andrii-zakurenyi) [inline README.md:94]: please remove this header

**Comment by mariiatuzovska** [comment]
The integration test should fail because it uses default branch as a source

**Review by sheldonhull — CHANGES_REQUESTED** [review]
Did running the integration test locally succeed? (Prebuilt and directions in developer.md) 

assuming failure on on test here is because the structure out the outputs changed while the app didn’t get published yet. 

Also you’ll want to follow that release notes process to ensure it generates a new changelog via changie.  See changie cli tool directions. That is what triggers the release process

**Comment by mariiatuzovska** [comment]
Integration tests ran successfully on local machine.

File: .changes/unreleased/Removed-20221017-155023.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+kind: Removed
+body: ::set-output action
```
[the remark this answers is not in the document]
Comment (sheldonhull) [reply .changes/unreleased/Removed-20221017-155023.yaml:2]: I see you updated the changelog file. Not sure if you did this manually. Changie batch and merge should get rid of the fragment files.

**Review by sheldonhull — CHANGES_REQUESTED** [review]
Minor changelog clarification and improved release notes please

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+## v2.0.1 - 2022-10-17
+
+### Changed
+
+- Set env variables by default
+
+### Removed
+
+- Remove deprecating ::set-output action
+- Remove setEnv flag
```
Comment (sheldonhull) [inline CHANGELOG.md:18]:
```suggestion
- BREAKING CHANGE: Remove setEnv flag
```

**Review by sheldonhull — CHANGES_REQUESTED** [review]
Highlight impact via breaking change note. Then we are good. 🎉

**Pull request merged**